code/modules/research/machinery/_production.dm

```diff
@@ -1,6 +1,8 @@
 /obj/machinery/rnd/production
 	name = "technology fabricator"
 	desc = "Makes researched and prototype items with materials and energy."
+	// Energy cost per full stack of materials spent. Material insertion is 40% of this.
+	active_power_usage = 20 KILO * BASE_MACHINE_ACTIVE_CONSUMPTION
```

Comment:
Isn't this value a bit too high? For autolathe you did just 25 * active consumption with the same comment so maybe that should apply here as well?

You can copy it's exact code for power consumption and make them match

Reply:
That was a brain fart. I multiplied by 20 instead of divide.